--- docs/source/conf.py.bak	2012-05-08 15:58:11.362541240 -0400
+++ docs/source/conf.py	2012-05-08 15:58:26.287448579 -0400
@@ -59,8 +59,8 @@
 # built documents.
 #
 def extract_version():
-    """extract pyzmq version from core/version.pyx, so it's not multiply defined"""
-    with open(os.path.join('..', '..', 'zmq', 'core', 'version.pyx')) as f:
+    """extract pyzmq version from core/version.py, so it's not multiply defined"""
+    with open(os.path.join('..', '..', 'zmq', 'core', 'version.py')) as f:
         line = f.readline()
         while not line.startswith("__version__"):
             line = f.readline()
